zero_bin/prover/src/cli.rs

```diff
@@ -0,0 +1,27 @@
+use clap::Args;
+
+const HELP_HEADING: &str = "Prover options";
+
+/// Represents the main configuration structure for the runtime.
+#[derive(Args, Clone, PartialEq, Eq, PartialOrd, Ord, Debug, Default)]
+pub struct CliProverConfig {
+    /// The log of the max number of CPU cycles per proof.
+    #[arg(short, long, help_heading = HELP_HEADING, default_value_t = 20)]
+    max_cpu_len_log: usize,
+    /// Number of transactions in a batch to process at once.
+    #[arg(short, long, help_heading = HELP_HEADING, default_value_t = 1)]
```

Comment:
non blocking (and technically unrelated to this PR), but I wonder if we shouldn't just set the default values to the close-to-optimal ones (i.e. 19 for CPU and 10 for batch size)? (Note that this would most likely require changing the ranges in tests with stdio mode)

Reply:
Yeah that makes sense to me